porch/repository/pkg/git/git.go

```diff
@@ -42,9 +42,12 @@ import (
 const (
 	refMain plumbing.ReferenceName = "refs/heads/main"
 	// TODO: support customizable pattern of draft branches.
-	refDraftPrefix = "refs/heads/drafts/"
-	refTagsPrefix  = "refs/tags/"
-	refHeadsPrefix = "refs/heads/"
+	refDraftPrefix        = "refs/heads/drafts/"
+	refProposedPrefix     = "refs/heads/proposed/"
```

Comment:
If we do need more metadata in future, we could attach git notes (or do what gerrit does, with a "metadata" branch)

Reply:
notes do seem like a useful technique!